virttest/qemu_devices/qdevices.py

```diff
@@ -1255,6 +1286,95 @@ def verify_unplug(self, out, monitor):
         return not self._is_attached_to_qemu(monitor)
 
 
+class QThrottleGroup(QObject):
+    """
+    throttle-group object.
+    attributes of throttle-group
+    """
+
+    KEY_MAP_TABLE = {"bps-read-max-length": "x-bps-read-max-length",
+                     "iops-read-max-length": "x-iops-read-max-length",
+                     "bps-read-max": "x-bps-read-max",
+                     "bps-total": "x-bps-total",
+                     "iops-total-max-length": "x-iops-total-max-length",
+                     "iops-total": "x-iops-total",
+                     "iops-write-max": "x-iops-write-max",
+                     "bps-write": "x-bps-write",
+                     "bps-total-max": "x-bps-total-max",
+                     "bps-write-max": "x-bps-write-max",
+                     "iops-size": "x-iops-size",
+                     "iops-read": "x-iops-read",
+                     "iops-write-max-length": "x-iops-write-max-length",
+                     "iops-write": "x-iops-write",
+                     "bps-total-max-length": "x-bps-total-max-length",
+                     "iops-read-max": "x-iops-read-max",
+                     "bps-read": "x-bps-read",
+                     "bps-write-max-length": "x-bps-write-max-length",
+                     "iops-total-max": "x-iops-total-max"}
+
+    def __init__(self, throttle_group_id, raw):
+        if not raw or len(raw) == 0:
+            # set default
+            logging.warning("Using unlimited value :%s" % throttle_group_id)
+            raw = "iops-total=0"
```

Comment:
Is there a default conf? I mean we don't explicitly set iops-total=0 here,  iops-total=0 will be used by qemu by default